doc/rst/language/spec/conversions.rst

```diff
@@ -136,17 +134,17 @@ converted to any integral type by representing ``false`` as 0 and
 Legal implicit conversions with numeric and boolean types may thus be
 summarized as follows:
 
-==================== ================= ================= ================ ================= ================= ====================
-\                                                                  **Destination Type**
--------------------- -------------------------------------------------------------------------------------------------------------
-**Source Type**      bool(\ :math:`t`) uint(\ :math:`t`) int(\ :math:`t`) real(\ :math:`t`) imag(\ :math:`t`) complex(\ :math:`t`)
-bool(\ :math:`s`)    all :math:`s,t`   all :math:`s,t`   all :math:`s,t`
-uint(\ :math:`s`)                      :math:`s \le t`   :math:`s < t`    all :math:`s,t`                     all :math:`s,t`
-int(\ :math:`s`)                       :math:`s \le t`   :math:`s \le t`  all :math:`s,t`                     all :math:`s,t`
-real(\ :math:`s`)                                                         :math:`s \le t`                     :math:`s \le t/2`
-imag(\ :math:`s`)                                                                           :math:`s \le t`   :math:`s \le t/2`
-complex(\ :math:`s`)                                                                                          :math:`s \le t`
-==================== ================= ================= ================ ================= ================= ====================
+==================== ================= ================ ================= ================= ====================
+\                                                **Destination Type**
+-------------------- -------------------------------------------------------------------------------------------
+**Source Type**      uint(\ :math:`t`) int(\ :math:`t`) real(\ :math:`t`) imag(\ :math:`t`) complex(\ :math:`t`)
+bool                 OK                OK
+uint(\ :math:`s`)    :math:`s \le t`   :math:`s < t`    all :math:`s,t`                     all :math:`s,t`
+int(\ :math:`s`)     :math:`s \le t`   :math:`s \le t`  all :math:`s,t`                     all :math:`s,t`
+real(\ :math:`s`)                                       :math:`s \le t`                     :math:`s \le t/2`
+imag(\ :math:`s`)                                                         :math:`s \le t`   :math:`s \le t/2`
+complex(\ :math:`s`)                                                                        :math:`s \le t`
+==================== ================= ================ ================= ================= ====================
```

Comment:
I prefer keeping with the style of the existing entries and instead of "OK" say "all :math:\`t\`".